### PR TITLE
refactor CLI for tekton

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

File: pkg/prow/prow.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 const (
 	Hook = "hook"
 
-	KnativeBuildAgent         = "knative-build"
-	KnativeBuildPipelineAgent = "knative-pipeline-run"
-	KubernetesAgent           = "kubernetes"
+	KnativeBuildAgent = "knative-build"
+	TektonAgent       = "takton"
```
Comment (abayer) [inline pkg/prow/prow.go]: ...`takton`? =)

Comment (jstrachan) [reply pkg/prow/prow.go]: fixed thanks!

**Pull request merged**